wikipron/config.py

```diff
@@ -35,6 +36,11 @@
 _PHONEMES_REGEX = r"/(.+?)/"
 _PHONES_REGEX = r"\[(.+?)\]"
 
+_TONES_REGEX = (
+    r"[\u02E5-\u02E9\u00b2-\u00b3\u00b9\u2074-\u2079"
```

Comment:
Sorry if I made this comment earlier but is it possible to replace these Unicode numerical codepoints with their human-readable versions? I believe it'll work just fine. (For instance, to obtain the printable version of \u02E5 I do:

    chr(0x02E5)